mobile/library/java/io/envoyproxy/envoymobile/utilities/StatsUtils.java

```diff
@@ -0,0 +1,30 @@
+package io.envoyproxy.envoymobile.utilities;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class for utilities around Envoy stats.
+ */
+public class StatsUtils {
+
+  /**
+   * Takes in a stats string from engine.dumpStats() and parses it into individual map entries.
+   * @param statsString, the string from dumpStats
+   * @return a map of stats entries, e.g. "runtime.load_success", "1"
+   */
+  public static Map<String, String> statsToList(String statsString) {
+    Map<String, String> stats = new HashMap<>();
+
+    String[] lines = statsString.split("\n");
```

Comment:
I don't know if it matters much (feel free to ignore), but the for loop can be rewritten more idiomatically as:

```
for (String line : statsString.split("\n")) {
   String[] keyValue = line.split(": ");
   // rest of the for loop
}
```